cmd/vclusterctl/cmd/platform/add/vcluster.go

```diff
@@ -60,5 +67,85 @@ vcluster platform add vcluster --project my-project --all
 
 // Run executes the functionality
 func (cmd *VClusterCmd) Run(ctx context.Context, args []string) error {
-	return cli.AddVClusterHelm(ctx, &cmd.AddVClusterOptions, cmd.GlobalFlags, args, cmd.Log)
+	localVClusters, err := cmd.localVClustersFromOptions(ctx, cmd.Log, cmd.GlobalFlags, args, cmd.AddVClusterOptions)
```

Comment:
WDYT of returning early in case if the length of localVClusters is empty, so we could skip the need to create platformClient and list all VCIs.

Reply:
Yes! Added